xrfeitoria/utils/anim/utils.py

```diff
@@ -120,59 +120,138 @@ def dump_humandata(
     )
 
 
+def batch_transform_to_camera_frame(global_orient, transl, pelvis, extrinsic):
```

Comment:
Maybe this function should be renamed to remove the `camera` in the function name.

Reply:
Solved in [`e6ee494` (#28)](https://github.com/openxrlab/xrfeitoria/pull/28/commits/e6ee49459ec673f979fef8f14e0bbebebdd8a453)